MAUI/Scheduler/month-view.md

```diff
@@ -322,6 +322,29 @@ public class MonthViewHeaderTemplateSelector : DataTemplateSelector
 N>
 * When using data template selector, performance issues occur as the conversion template views take time within the framework.
 
+## Month Navigation direction
+
+MonthView of SfScheduler can be navigated in both horizontal and vertical direction. You can change the direction of navigation through `NavigationDirection` property of MonthViewSettings in SfSchedule, by default Month navigation direction is `Horizontal`. If the direction is  `Vertical` the `ViewHeader` will be sticky.
```

Comment:
The `MonthView` of `SfScheduler` can be navigated in both horizontal and vertical directions. You can change the direction of navigation through the `NavigationDirection` property of `MonthViewSettings` in `SfScheduler`. By default, the navigation direction is `Horizontal`.

Reply:
Updated the changes